controllers/funding/index.js

```diff
@@ -138,13 +138,31 @@ module.exports = (pages, sectionPath, sectionId) => {
                     req.flash('formErrors', result.array());
                     req.flash('formValues', req.body);
                     req.session.save(() => {
-                        res.redirect(req.baseUrl + freeMaterials.path + '#your-details');
+                        // build a redirect URL based on the route, the language of items,
+                        // and the form anchor, so the user sees the form again via JS
+                        let returnUrl = req.baseUrl + freeMaterials.path;
+                        let formAnchor = '#your-details';
+                        let langParam = '?lang=';
+                        let redirectUrl = returnUrl;
+
+                        // add their langage choice (if valid)
+                        let langChoice = req.body['languageChoice'];
```

Comment:
Any reason for `req.body['languageChoice']` over `req.body.languageChoice`?

Reply:
No – good call, have changed to match other uses!